src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/util/SalesforceSourceConstants.java

```diff
@@ -35,17 +35,15 @@
   public static final String PROPERTY_PK_CHUNK_ENABLE_NAME = "enablePKChunk";
   public static final String PROPERTY_CHUNK_SIZE_NAME = "chunkSize";
   public static final String PROPERTY_PARENT_NAME = "parent";
+  public static final String PROPERTY_QUERY_SPLITS = "querySplits";
 
-  public static final String PROPERTY_WHITE_LIST = "whiteList";
-  public static final String PROPERTY_BLACK_LIST = "blackList";
+  public static final String PROPERTY_ALLOW_LIST = "allowList";
```

Comment:
you would also need to update https://github.com/data-integrations/salesforce/blob/develop/widgets/SalesforceMultiObjects-batchsource.json#L64. Suggest doing renaming changes in a separate PR as it's unrelated to this fix.

Reply:
Thanks. I also see similar terms in the UI as well as in a couple other files. I did not change that because I wasn't sure if it wouldn't break anything. Will drop these changes for now